cli/azd/cmd/testdata/TestUsage-azd-deploy.snap

```diff
@@ -11,7 +11,7 @@ Usage
 Flags
         --all                 	: Deploys all services that are listed in azure.yaml
     -e, --environment string  	: The name of the environment to use.
-        --from-package string 	: Deploys the application from an existing package.
+        --from-package string 	: Deploys a single service at a time from an existing package at the provided path. Supports both generated zip file packages or local container builds created by azd package or other means.
```

Comment:
Just curious, does this sound any better?

Deploys the packaged service located at the provided path. Supports zipped file packages (file path) or container images (image tag).